src/Grpc.Net.Client/GrpcClient.cs

```diff
@@ -37,7 +37,7 @@ public static class GrpcClient
         /// <param name="httpClient">The <see cref="HttpClient"/>.</param>
         /// <param name="loggerFactory">The <see cref="ILoggerFactory"/>.</param>
         /// <returns>A gRPC client.</returns>
-        public static TClient Create<TClient>(HttpClient httpClient, ILoggerFactory? loggerFactory = null) where TClient : ClientBase<TClient>
+        public static TClient Create<TClient>(HttpClient httpClient, ILoggerFactory? loggerFactory = null) where TClient : class
```

Comment:
Add a comment here (and in client factory) explaining why TClient is class and not ClientBase/LiteClient. I worry about someone in the future coming in and attempting to change it without understanding the problem